### PR TITLE
[docs] Classic/Standalone apps: Update the documentation

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Fixes ENG-5388

# How

This PR updates to documentation to use 
- Consistent verbiage, and Terminal component for copy/paste commands. 
- It also adds a new sub-section called `Uploading your keystore after publishing the app` and replaces the he big wall of text that was used as a "Note" before.

Simplifies other sections in the doc too.

# Test Plan

The changes have been tested by running docs locally.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/classic/building-standalone-apps.md [hunk 1]
```diff
@@ -44,62 +43,84 @@ to launch Ubuntu at least once. After that, use an Admin powershell to run:
...
 If you don't know what this means, let us handle it! :)
 
-  1) Let Expo handle the process!
-  2) I want to upload my own keystore!
+1) Let Expo handle the process!
+2) I want to upload my own keystore!
 ```
 
-> **Note:** If you choose the first option and later decide to upload your own keystore, we currently offer an option to clear your current Android keystore from our build servers by running `expo build:android --clear-credentials.` **This is irreversible, so only run this command if you know what you are doing!** You can download a backup copy of the keystore by running `expo fetch:android:keystore`. **If you do not have a local copy of your keystore , you will be unable to publish new versions of your app to the Play Store**. Your only option would be to generate a new keystore and re-upload your application as a new application. You can learn more about how code signing and keystores work [in the Android documentation](https://developer.android.com/studio/publish/app-signing.html).
+If you choose the first option, it is **strongly recommend** that you run the following command after the build process is complete and backup the keystore to a safe location:
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:95]:
```suggestion
If you choose the first option, it is **strongly recommended** that you run the following command after the build process is complete. After running it, backup the keystore in a safe location:
```

File: docs/pages/classic/building-standalone-apps.md [hunk 2]
```diff
@@ -44,62 +43,84 @@ to launch Ubuntu at least once. After that, use an Admin powershell to run:
...
+
+<Terminal cmd={['$ expo fetch:android:keystore']} />
+
+Once you submit the app to the Google Play Store, all future updates to that app **must** be signed with the same keystore that the Play Store initially accepted. If you do not have a local copy of your keystore, you will be unable to publish new versions of your app to the Play Store.
+
+For any reason you delete your project or clear your credentials in the future, you will not be able to submit any updates to your app without providing the same keystore. Your only option left is to generate a new keystore and re-upload your application as a new application.
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:101]:
```suggestion
If for any reason you delete your project or clear your credentials in the future, you will not be able to submit any updates to your app without providing the same keystore. If you're unable to retrieve the keystone, your only option left is to generate a new keystore and re-upload your application as a new application.
```

File: docs/pages/classic/building-standalone-apps.md [hunk 3]
```diff
@@ -44,62 +43,84 @@ to launch Ubuntu at least once. After that, use an Admin powershell to run:
...
+### Building for iOS
 
-You can build standalone apps for iOS with two different types, an archive (`expo build:ios -t archive`) or simulator (`expo build:ios -t simulator`) build.
-With the simulator build, you can test your standalone app on a simulator.
-If you want to publish your app to the store or distribute it with tools like TestFlight, you have to use the archive.
+There are two types of build you can generate when building a standalone app for iOS: archive and simulator.
 
-When building for iOS, you are given a choice of letting Expo create the necessary credentials for you,
-while still having a chance to provide your own overrides. Your Apple ID and password are used locally and are never saved on Expo's servers.
+Archive build is used for publishing your app to the Apple App Store or distributing it with services like TestFlight. To build an archive, run the command:
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:115]:
```suggestion
Archive builds are used for submitting your app to the Apple App Store and distributing it with features like TestFlight. To build an iOS app archive, run:
```

File: docs/pages/classic/building-standalone-apps.md [hunk 4]
```diff
@@ -44,62 +43,84 @@ to launch Ubuntu at least once. After that, use an Admin powershell to run:
...
+
+<Terminal cmd={['$ expo build:ios -t archive']} />
+
+Simulator build is used to test the standalone app on an iOS simulator. To build it, run the command:
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:119]:
```suggestion
Simulator builds are used to test a standalone app in an iOS simulator. To build one, run:
```

File: docs/pages/classic/building-standalone-apps.md [hunk 5]
```diff
@@ -44,62 +43,84 @@ to launch Ubuntu at least once. After that, use an Admin powershell to run:
...
+
+<Terminal cmd={['$ expo build:ios -t simulator']} />
+
+Building for iOS, you can choose the option to let Expo create necessary credentials. You still have the choice to provide your own credentials. In this process, your Apple ID and password are used locally and are never saved on Expo's servers.
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:123]:
```suggestion
When building for iOS, you can choose to let Expo create any necessary credentials that your app will need. However, you'll still have the choice to provide your own credentials if you'd like. During this process, Expo CLI will use your Apple ID and password. Your authentication information is only used locally on your computer and is never sent to Expo's servers.
```

File: docs/pages/classic/building-standalone-apps.md [hunk 6]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
   I will provide all the credentials and files needed, Expo does limited validation
 ```
 
-Unless you're very familiar with iOS credentials already, it's best to let Expo handle the creation & management of all your credentials for you. If you'd like to know more about iOS credentials, we've written a guide with everything you need to know [here](/app-signing/app-credentials.md).
+If you are unfamiliar with iOS credentials, it is recommended to let Expo handle the creation & management of credentials. To know more about them, refer to the [App Credentials documentation](/app-signing/app-credentials).
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:148]:
```suggestion
If you are unfamiliar with iOS credentials, it is recommended to let Expo handle the creation & management of credentials. To learn more about them, refer to the [App Credentials documentation](/app-signing/app-credentials).
```

File: docs/pages/classic/building-standalone-apps.md [hunk 7]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
...
+If you are familiar with iOS credentials and plan to provide your own, use the [Apple Developer Portal](https://developer.apple.com/account/resources/certificates/list) to create them.
 
 #### Enterprise distribution
 
-The Expo build service supports both normal App Store distribution as well as enterprise distribution. To use the latter, you must be a member of the ["Apple Developer Enterprise Program"](https://developer.apple.com/programs/enterprise/). Only normal Apple Developer accounts can build apps that can be submitted to the Apple App Store, and only enterprise developer accounts can build apps that can be distributed using enterprise distribution methods. When you call `expo build:ios`, you just need to choose the correct team, it will be labeled `(In-House)`.
+To build and distribute your app using Apple's Enterprise distribution, it is required that you are a member of the [Apple Developer Enterprise Program](https://developer.apple.com/programs/enterprise/).
+
+Expo Build Service supports Enterprise distribution. When you call `expo build:ios`, you just need to choose the correct team that is labeled with `(In-House)`.
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:156]:
```suggestion
Expo's build service supports Enterprise distribution. When you run `expo build:ios`, you'll need to choose the correct team that is labeled with `(In-House)`.
```

File: docs/pages/classic/building-standalone-apps.md [hunk 8]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
...
 
 #### Adhoc distribution
 
-> Since this requires you to provide credential overrides, this feature should only be used if you're comfortable with iOS credentials.
+> This distribution requires you to override and use your own credentials credential overrides. Use this distribution if you know how to generate iOS credentials.
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:160]:
```suggestion
> This distribution method requires you to override your own credentials. Use this distribution if you know how to generate iOS credentials.
```

File: docs/pages/classic/building-standalone-apps.md [hunk 9]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
...
+
+One of the easiest ways to test your standalone app is using a [simulator build](#building-for-ios) and then using TestFlight for your physical device for testing. However, if you want to build and install an IPA directly on a physical device through Xcode, follow the steps below to generate one using Expo CLI:
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:162]:
```suggestion
One of the easiest ways to test your standalone app is using a [simulator build](#building-for-ios) and then using TestFlight for your physical device for testing. However, if you want to build and install an app directly on a physical device through Xcode, follow the steps below to generate one using Expo CLI:
```
Another styleguide point. We should say "app" when referring to a .ipa .apk or .aab file. We should use ".ipa" when we want to specifically talk about the extension.

File: docs/pages/classic/building-standalone-apps.md [hunk 10]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
...
+
+- Generate preliminary certificate such as app identifier and distribution certificate by running the command:
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:164]:
```suggestion
- Generate preliminary certificates such as app identifier and distribution certificate by running the command:
```

File: docs/pages/classic/building-standalone-apps.md [hunk 11]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
...
+If you use a Push Notifications Certificate and want to switch to a [Push Notifications Key](app-signing/app-credentials/#push-notification-keys), start the build with `--clear-push-cert` option. Expo will remove legacy certificates from its servers and generate a Push Notifications Key for you.
 
 ## 4. Wait for it to finish building
 
-When one of our building machines is free, it'll start building your app. You can check how long you'll wait on the [Turtle status](https://expo.dev/turtle-status) site. We'll print a url you can visit (such as `expo.dev/builds/some-unique-id`) to watch your build progress and access the build logs. Alternatively, you can check up on it by running `expo build:status`. When it's done, you'll see the url to your app file - an **.apk**, **.aab** (both Android), or **.ipa** (iOS) file. Copy and paste the link into your browser to download the file.
+Expo Build Service uses building machines to build your app. The build process starts as soon as one of the machines is free and ready to be used. To check how long the waiting time for your build is, visit [Turtle status](https://expo.dev/turtle-status) site.
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:190]:
```suggestion
Expo's build service uses cloud services to build your app. The build process starts as soon as one of the machines is free and ready to be used. To check how long the waiting time for your build is, visit [Turtle status](https://expo.dev/turtle-status) site.
```

File: docs/pages/classic/building-standalone-apps.md [hunk 12]
```diff
@@ -124,65 +145,85 @@ Note: Expo does not keep your Apple ID or your Apple ID password.
...
+
+Expo CLI prints a URL with a unique ID, such as `expo.dev/builds/some-unique-id`, for you to visit and check your build progress. You can also access build logs on this URL.
```
Comment (jonsamp) [inline docs/pages/classic/building-standalone-apps.md:192]:
```suggestion
Expo CLI prints a URL that you can visit and check your build for its progress and logs.
```

**Review by jonsamp — APPROVED** [review]
Thanks for updating this doc. I left a lot of comments about the wording, but overall this is the right direction. Once you review them/commit them, let's merge this PR!

File: docs/pages/classic/building-standalone-apps.md [hunk 13]
```diff
@@ -44,62 +43,84 @@ to launch Ubuntu at least once. After that, use an Admin powershell to run:
...
 - The `ios.buildNumber` and `android.versionCode` distinguish different binaries of your app. Make sure to increment these for each build you upload to the App Store or Google Play Store.
 
-There are other options you might want to add to **app.json**. We have only covered what is
-required. For example, some people like to configure their own build number, linking scheme, and
-more. We highly recommend you read through [Configuration with app.json / app.config.js](/workflow/configuration.md) for the
-full spec. This is also your last chance to double check our [recommendations](/distribution/app-stores.md)
-for App Store metadata.
+There are other options you can add to **app.json**. For example, some developers like to configure their own build number, linking scheme, etc. We recommend you to go through [Configuration with app.json/app.config.js](/workflow/configuration) for the full specification. At this step, also check [Deploying to App Stores](/distribution/app-stores) documentation. It provides details on what type of metadata is required for App stores.
```
[the remark this answers is not in the document]
Comment (amandeepmittal) [reply docs/pages/classic/building-standalone-apps.md:57]: Duly noted. I will add this to the style guide.

**Comment by amandeepmittal** [comment]
Thanks for your in-depth review @jonsamp! Learned quite a few points on maintaining a good, understandable level of the copy.

**Pull request merged**